docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -338,6 +341,52 @@ For some performance-critical DL operations, the CPU plugin uses third-party lib
 Optimization guide
 ###########################################################
 
+Multi-Threading Optimization
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+CPU inference will infer an input or multiple inputs in parallel on multiple CPU processors. 
```

Comment:
Can we update all invocations of `processor` to `logical processor`?

Reply:
updated